services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/AbstractRegistrationService.java

```diff
@@ -159,6 +276,38 @@ public final void setTenantInformationService(final TenantInformationService ten
         return assertRegistration(tenantId, deviceId, gatewayId, NoopSpan.INSTANCE);
     }
 
+    /**
+     * Add a new edge device in the process of auto-provisioning of not yet registered edge devices sending data
```

Comment:
We always use active form present tense for JavaDocs.
 
> Adds a new edge ...

Reply:
I know but missed that one. Thanks for spotting.